src/main/resources/global/91-init-share.sh

```diff
@@ -10,11 +10,21 @@ ALFRESCO_PORT=${ALFRESCO_PORT:-8080}
 ALFRESCO_PROTOCOL=${ALFRESCO_PROTOCOL:-http}
 ALFRESCO_CONTEXT=${ALFRESCO_CONTEXT:-alfresco}
 
+ALFRESCO_INTERNAL_HOST=${ALFRESCO_INTERNAL_HOST:-alfresco}
```

Comment:
Can we default this to $ALFRESCO_HOST instead? Or is that a bad idea?

Reply:
Imo we want to keep these 2 sets of variables independent.
In single-setup cases, one would leave defaults for INTERNAL variables and would set ALFRESCO_HOST to the DNS name.

Reply:
Oh okay, I understand.